api/openapi.yaml

```diff
@@ -738,335 +738,478 @@ paths:
+        - $ref: "#/components/parameters/entitlementId"
+        - $ref: "#/components/parameters/includeDeleted"
+        # - $ref: "#/components/parameters/queryLimit"
+        # - $ref: "#/components/parameters/queryOffset"
+        - name: orderBy
+          in: query
+          required: false
+          description: Order by field
+          schema:
+            type: string
+            enum:
+              - id
+              - createdAt
+              - updatedAt
+            default: updatedAt
       responses:
         "200":
-          description: List of ledger grants.
+          description: List of grants.
           content:
             application/json:
               schema:
                 type: array
                 items:
-                  $ref: "#/components/schemas/LedgerGrantResponse"
-        "400":
-          $ref: "#/components/responses/BadRequestProblemResponse"
+                  $ref: "#/components/schemas/EntitlementGrant"
         "401":
           $ref: "#/components/responses/UnauthorizedProblemResponse"
         default:
           $ref: "#/components/responses/UnexpectedProblemResponse"
-  /api/v1/ledgers/{ledgerID}/grants:
-    get:
-      operationId: listLedgerGrantsByLedger
-      summary: List ledger grants
+    post:
+      operationId: createGrant
+      summary: Create grant
       description: |
-        List ledger grants for a specific ledger.
+        Create a grant for an entitlement.
       tags:
         - Entitlements (Experimental)
       parameters:
-        - $ref: "#/components/parameters/ledgerID"
-        - $ref: "#/components/parameters/ledgerQueryLimit"
-        - $ref: "#/components/parameters/ledgerIncludeVoids"
+        - $ref: "#/components/parameters/subjectIdOrKey"
+        - $ref: "#/components/parameters/entitlementId"
+      requestBody:
+        description: The grant to create.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/EntitlementGrantCreateInput"
       responses:
-        "200":
-          description: List of ledger grants created.
+        "201":
+          description: Grant created.
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/LedgerGrantResponse"
+                $ref: "#/components/schemas/EntitlementGrant"
         "400":
           $ref: "#/components/responses/BadRequestProblemResponse"
         "401":
           $ref: "#/components/responses/UnauthorizedProblemResponse"
+        "501":
+          $ref: "#/components/responses/NotImplementedProblemResponse"
         default:
           $ref: "#/components/responses/UnexpectedProblemResponse"
-    post:
-      operationId: createLedgerGrant
-      summary: Create a grant on a specific ledger.
+  /api/v1/subjects/{subjectIdOrKey}/entitlements/{entitlementId}/balance:
```

Comment:
Thinking loudly: if we have one entitlement per feature, we can allow `entitlementIdOrFeatureKey` param. Which again helps to check balance on UI side.

Reply:
I think the idea was to push till later that decision